frontend/stories/components/search/SearchListItem.stories.tsx

```diff
@@ -37,22 +37,22 @@ const meta: Meta<typeof Item> = {
         agency_phone_number: null,
         applicant_eligibility_description: null,
         applicant_types: null,
-        award_ceiling: 200000000,
-        award_floor: 1,
+        awardCeiling: 200000000,
```

Comment:
the attributes in this file should remain in snake case as they represent the data shape that is coming back from the API